pyproject.toml

```diff
@@ -18,9 +18,9 @@ platforms = ["win-64"]
 Dynamic-DB = { path = ".", editable = true }
 
 [tool.pixi.tasks]
-poc = "uvicorn POC.api.main:app --reload"
-test_pre_commit = "pre-commit run --all-files"
-test_poc_mypy = "mypy --check-untyped-defs --strict"
+poc = "uvicorn POC.api.main:app"
```

Comment:
removed reload for default environment and added reload to test environment.

Reply:
Nice, great that you know how to leverage the different commands 👍